onmt/opts.py

```diff
@@ -249,8 +238,8 @@ def preprocess_opts(parser):
     group.add('--valid_align', '-valid_align', default=None,
               help="Path(s) to the validation src-tgt alignment")
 
-    group.add('--src_dir', '-src_dir', default="",
-              help="Source directory for image or audio files.")
+    group.add('--src_dir', '-src_dir', default="", action=DeprecateAction,
+              help="Source directory for image files.")
 
     group.add('--save_data', '-save_data', required=True,
```

Comment:
Is this one still used since everything can be written next to the checkpoints at train.py execution?

Reply:
This also used as the directory to save the sample files even if we written everything into the checkpoints.